podget

```diff
@@ -281,7 +281,7 @@ LOG_COMPLETE=done
 WGET_BASEOPTS="-c -nH"
 
 # Most Recent
-# 0  == download all new items.
+# 0  == download all new items. Careful! this can swallow whole bandwidth with ancient feeds.
```

Comment:
OK let me repeat myself.  This has no place in this pull request.  It is a separate non-OS-specific issue.  If you care open a new Issue to discuss it, go ahead.